genie-core/src/main/java/com/netflix/genie/core/jpa/entities/JobRequestEntity.java

```diff
@@ -159,8 +159,12 @@ protected String getClusterCriterias() {
      *
      * @param clusterCriterias The cluster criterias.
      */
-    protected void setClusterCriterias(@NotBlank final String clusterCriterias) {
-        this.clusterCriterias = clusterCriterias;
+    protected void setClusterCriterias(final String clusterCriterias) {
+        if (StringUtils.isEmpty(clusterCriterias)) {
```

Comment:
You could simplify this using the ternary: `this.clusterCriterias = StringUtils.isBlank(clusterCriterias) ? EMPTY_JSON_ARAY : clusterCriterias;`